src/docs/implementations/dynatrace.adoc

```diff
@@ -4,25 +4,170 @@ Jon Schneider <jschneider@pivotal.io>
+
+*Configuration with URI and API token*
+
+If no local OneAgent is running on the machine or the metrics should be sent to a specific endpoint, the Dynatrace v2 exporter can be configured with a Dynatrace environment URL and an https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/[API token].
+The https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/[API token] must have the https://www.dynatrace.com/support/help/shortlink/api-authentication#token-permissions["Ingest metrics"] (`metrics.ingest`) permission set.
+It is recommended to limit scope to this permission.
+
+Please also ensure that the endpoint URL is complete, i. e. it contains the path `/api/v2/metrics/ingest` on SaaS and managed deployments, or `/metrics/ingest` for OneAgent endpoints.
+
+*Properties available in the v2 exporter*
+
+When using the https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/[Dynatrace metrics API v2] additional optional properties can be set:
+
+[source,yml]
+----
+management.metrics.export.dynatrace:
+    # required to use the properties below
+    api-version: v2
+
+    # required if not using a local OneAgent endpoint.
+    uri: https://your-environment-id.live.dynatrace.com/api/v2/metrics/ingest
+    # or https://your-domain/e/{your-environment-id}/api/v2/metrics/ingest on Managed
+
+    # We do *not* recommend putting the token or URI into the .yml file in clear text.
+    # We suggest using environment variables, e. g. api-token: ${ENV_VAR_DT_INGEST_TOKEN}
+    api-token: YOUR_TOKEN
+
+    # Sets a prefix that is prepended to each exported metric key.
+    metric-key-prefix: your.key.prefix
+
+    # If set to true and a local OneAgent is running, exports OneAgent metadata to the specified Dynatrace endpoint (default: false)
+    enrich-with-one-agent-metadata: true
+
+    # Sets an arbitrary number of key-value pairs as default dimensions.
+    # Micrometer tags will overwrite these dimensions, if the same key appears twice.
+    # Each exported metric will contain these dimensions.
+    default-dimensions:
+        key1: "value1"
+        key2: "value2"
+----
+
+These properties can also be set in code by overwriting the respective functions.
+
+=== API v1 (Legacy)
+
+When the apiVersion is configured to "v1", the registry will send data using the https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v1/custom-metrics/[Dynatrace Timeseries API v1 for custom metrics].
+The reported metrics will be assigned to https://www.dynatrace.com/support/help/dynatrace-api/environment-api/topology-and-smartscape/custom-device-api/report-custom-device-metric-via-rest-api/[custom devices] in Dynatrace.
+If no apiVersion is specified, it will default to "v1".
+
+For the v1 API, do not specify the ingest path, but only the base URL of your environment, e. g. `uri: https://your-environment-id.live.dynatrace.com`
```

Comment:
```suggestion
For the v1 API, do not specify the ingest path, but only the base URL of your environment, e. g. `uri: https://{your-environment-id}.live.dynatrace.com`
```

I think the placeholders are quite nice because this makes it super clear that this can't be used as a default for demo purposes or similar since it will throw.

Reply:
Also, this is consistent with the Dynatrace Doc. I somehow thought it might be misleading, given that the environment variable substitution looks so similar (`${TEXT}` vs. `{TEXT}`).